test/common_test.lua

```diff
@@ -214,6 +216,43 @@ function g.test_functions_errors()
             }),
         }}
     )
+
+end
+
+function g.test_http_event_removing()
+    local config = {{
+        filename = 'extensions/config.yml',
+        content = yaml.encode({
+            functions = {x = {
+                module = 'extensions.box',
+                handler = 'cat',
+                events = {{
+                    http = { path = '/cat', method = 'GET' }
+                }}
+            }}
+        })
+    }, {
+        filename = 'extensions/box.lua',
+        content = "return { cat = function() return { status = 200, body = \"Meow\" } end}"
+    }}
+
+    set_sections(config)
+    local res = http.get(("localhost:%s/cat"):format(g.cluster.main_server.http_port))
```

Comment:
Move it to a local variable

Reply:
???

Reply:
It's used twice in the function. It looks reasonable to reuse it.